pagopa/io-app
# chore(Messaggi a valore legale): [IAMVL-51] Support two lines filename and preserve extension

## Short description
Use up to two lines for attachment's filename while preserving the extension.

## How to test
Use https://github.com/pagopa/io-dev-api-server/pull/125 to get a very long filename as a PNG attachment.

Notice the results below:

| Before | After |
| -------|-------|
| ![Screenshot 2022-01-24 at 17 07 27](https://user-images.githubusercontent.com/2094604/150822420-715c2a61-0d62-4590-9af5-951553954637.png) | ![Screenshot 2022-01-24 at 17 03 07](https://user-images.githubusercontent.com/2094604/150822468-1dc85166-b0d4-4944-8473-f7362e504345.png) |
